style.css

```diff
@@ -166,104 +139,586 @@ header {
+  #start-collab-btn {
+    width: 50%;
+  }
+
+  .main-end {
+    display: none;
+  }
+
+  /* project section */
+
+  #projects {
+    background-color: #fff;
+    display: grid;
+    grid-template-rows: 6.41fr 2fr 1fr 1fr 2fr 3.16fr 3.1fr;
+    grid-template-columns: 2.66fr 1fr 1fr 1fr 1.66fr 1fr;
+    grid-template-areas:
+      "project-title project-1 project-1 project-1 project-1 project-1"
+      "project-2     bg-1      bg-1      bg-1      project-6 project-6"
+      "project-2     bg-1      bg-1      bg-1      project-6 project-6"
+      "project-2     bg-1      bg-1      bg-1      project-5 project-5"
+      "project-3     project-3 project-3 .         project-5 project-5"
+      "project-3     project-3 project-3 bg-2      bg-2      bg-2"
+      "bg-3          bg-3      bg-3      project-4 project-4 .";
+    margin-bottom: 12.5rem;
+  }
+
+  .project {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    background-color: #fff;
+  }
+
+  .project-img {
+    display: block;
+    width: 100%;
+    object-fit: cover;
+    margin-top: 0;
+    margin-left: 0;
+  }
+
+  .project-overlay {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 0;
+    background: rgba(26, 34, 54, 0.95);
+    opacity: 0;
+    overflow: hidden;
+    transition: 0.5s ease;
+  }
+
+  .project:hover {
+    transition: all 0.5s ease-in;
+    transform: scale(1.01);
+  }
+
+  .project:hover .project-overlay {
+    height: 75%;
+    opacity: 1;
+  }
+
+  #project-1 {
+    grid-area: project-1;
+  }
+
+  #project-2 {
+    grid-area: project-2;
+    position: relative;
+  }
+
+  #project-2 .project-img {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+
+  #project-3 {
+    grid-area: project-3;
+  }
+
+  #project-4 {
+    grid-area: project-4;
+  }
+
+  #project-5 {
+    grid-area: project-5;
+  }
+
+  #project-6 {
+    grid-area: project-6;
+  }
+
+  #bg-1 {
+    grid-area: bg-1;
+    background-image: url(Images/icons/background-1.svg);
+    background-size: 100% 100%;
+  }
+
+  #bg-2 {
+    grid-area: bg-2;
+    background-image: url(Images/icons/background-2.svg);
+  }
+
+  #bg-3 {
+    grid-area: bg-3;
+    background-image:
+      url(Images/icons/background-3.svg),
+      url(Images/icons/image-geometry_4.svg);
+    background-repeat: no-repeat, no-repeat;
+    background-position-x: left, right;
+    background-position-y: top, top;
+    background-size: 50% 50%, 50% 50%;
+  }
+
+  .project-head {
+    padding-left: 6.25rem;
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: top;
+    background-image: url(Images/icons/image_dots.svg);
+  }
+
+  #project-title {
+    grid-area: project-title;
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+    writing-mode: vertical-lr;
+    transform: rotate(180deg);
+  }
+
+  /* about me section */
+
+  .about {
+    flex-direction: row-reverse;
+    padding: 6.25rem;
+
+    /* gap: 2rem; */
+  }
+
```

Comment:
[OPTIONAL] Please remove unwanted commented code from your codebase because it's your codebase make messy if you want to check your previous code then check GitHub commit history :+1: